readme.md

```diff
@@ -1097,3 +1098,153 @@ skip-csproj-packagereference: true
 ```
 
 </details>
+
+## Mgmt plane configurations
+
+The mgmt .Net SDK requires the resources to be generated hierarchical, therefore mgmt generator makes quite a few modification on the code model inout from the modelerfour. To better recognize the hierarchical structure of resources, the mgmt generator will generate everything from the point of view of request paths, instead of operation groups. Therefore quite a few new configurations are introduced to tweak the behavior how we identify the resource classes as well as the hierarchy.
+
+### Changing resource data
+
+In general, mgmt generator needs to determine which request path corresponds to a resource class first, and then generates the hierarchical structure based on that. By default, the generator will first check if the request path has even segments starting from the last `providers` segment (for instance, `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/availabilitySets/{name}` meets this requirement, but `/{scope}/providers/Microsoft.Authorization/policyAssignments` does not), then check if it has a `GET` request, and its response schema is compatible to ARM's resource definition - it must have `id`, `type` and `name`. If all these conditions are met, the request path is identified as a resource class path, and the response schema of the `GET` request will be the corresponding `ResourceData`.
+
+You can change this behavior by using the configuration `request-path-to-resource-data`, which is a dictionary from request paths to a name of schema, for instance
+
+```
+request-path-to-resource-data:
+  /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/availabilitySets/{name}: Something
```

Comment:
Any rule for 'Something'?   Start with capital letter?

Reply:
what if i input  /{scope}/providers/Microsoft.Authorization/policyAssignments: Something, does it still work?

Reply:
Let me add some details